frontend/lib/norent/data/state-localized-resources.tsx

```diff
@@ -0,0 +1,40 @@
+import { USStateChoice } from "../../../../common-data/us-state-choices";
+import { t } from "@lingui/macro";
+import { LocalizedOutboundLinkProps } from "../../ui/localized-outbound-link";
+
+export type StateLocalizedResources = Partial<
+  {
+    [k in USStateChoice]: LocalizedOutboundLinkProps[];
+  }
+>;
```

Comment:
This might be overkill since we only have one entry (NY), but I figure it's just as easy as hard-coding an `if` statement but makes it super easy to add resources for other states if we ever need to (it seems like the copy may have been crafted with this in mind).

Reply:
Nice! that looks good to me